website/sidebars.js

```diff
@@ -1,99 +1,157 @@
-        { type: "doc", id: "sdk-reference/dart", label: "Dart (Flutter)" },
-        { type: "doc", id: "sdk-reference/python", label: "Python" },
-        { type: "doc", id: "sdk-reference/go", label: "Go" },
-        { type: "doc", id: "sdk-reference/php", label: "PHP" },
-        { type: "doc", id: "sdk-reference/ruby", label: "Ruby" },
-        { type: "doc", id: "sdk-reference/elixir", label: "Elixir" },
-        { type: "doc", id: "sdk-reference/cpp", label: "C++" },
+      'SDK References': [
+        { type: 'doc', id: 'sdk-reference/overview', label: 'Overview' },
+        { type: 'doc', id: 'sdk-reference/dotnet', label: '.Net, .Net Core' },
+        { type: 'doc', id: 'sdk-reference/java', label: 'Java' },
+        { type: 'doc', id: 'sdk-reference/js', label: 'JavaScript' },
+        { type: 'doc', id: 'sdk-reference/js-ssr', label: 'JavaScript (SSR)' },
+        { type: 'doc', id: 'sdk-reference/react', label: 'React' },
+        { type: 'doc', id: 'sdk-reference/node', label: 'Node.js' },
+        { type: 'doc', id: 'sdk-reference/android', label: 'Android (Java)' },
+        {
+          type: 'doc',
+          id: 'sdk-reference/kotlin',
+          label: 'Kotlin Multiplatform',
+        },
+        { type: 'doc', id: 'sdk-reference/ios', label: 'iOS (Swift)' },
+        { type: 'doc', id: 'sdk-reference/dart', label: 'Dart (Flutter)' },
+        { type: 'doc', id: 'sdk-reference/python', label: 'Python' },
+        { type: 'doc', id: 'sdk-reference/go', label: 'Go' },
+        { type: 'doc', id: 'sdk-reference/php', label: 'PHP' },
+        { type: 'doc', id: 'sdk-reference/ruby', label: 'Ruby' },
+        { type: 'doc', id: 'sdk-reference/elixir', label: 'Elixir' },
+        { type: 'doc', id: 'sdk-reference/cpp', label: 'C++' },
       ],
-      "Integrations": [
-        { type: "doc", id: "integrations/overview", label: "Overview" },
-        { type: "doc", id: "integrations/slack", label: "Slack" },
-        { type: "doc", id: "integrations/jira", label: "Jira Cloud Plugin" },
-        { type: "doc", id: "integrations/trello", label: "Trello Power-Up" },
-        { type: "doc", id: "integrations/monday", label: "monday.com" },
-        { type: "doc", id: "integrations/datadog", label: "Datadog" },
-        { type: "doc", id: "integrations/zapier", label: "Zapier Zap" },
-        { type: "doc", id: "integrations/zoho-flow", label: "Zoho Flow" },
-        { type: "doc", id: "integrations/github", label: "GitHub Action" },
-        { type: "doc", id: "integrations/circleci", label: "CircleCI Orb" },
-        { type: "doc", id: "integrations/bitbucket", label: "Bitbucket Pipe" },
-        { type: "doc", id: "integrations/bitrise", label: "Bitrise Step" },
-        { type: "doc", id: "integrations/terraform", label: "Terraform" },
-        { type: "doc", id: "integrations/amplitude", label: "Amplitude" },
-        { type: "doc", id: "integrations/vscode", label: "Visual Studio Code" },
+      Integrations: [
```

Comment:
This is odd compared to the others. Why loosing the ' or "?

Reply:
Yeah, there are sometimes single quotes, sometimes nothing like here. Seems like it is still valid without single quotes if it is a single word. But if it has space in it, then the quotes are required.